specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -233,6 +233,67 @@
             }
           }
         }
+      },
```

Comment:
post is not recommended way to create resources in Azure. we are even going to make create of subscriptions as a PUT. so a billingProfile creation must be a PUT

Reply:
unfortunately, we are calling into the commerce system which does not allow us to specify the Id for the object to create. This prevents us from using PUT. In the case of subscriptions, i believe the caller is allowed to specify the Id of the subscription to create.

Reply:
This will have to be approved by Mark Simms. Kris and Mark had worked with billing RP to change the POST for subscription creation to PUT. I mentioned this to @KrisBash  and he said he will circle on this with Mark. Please follow up with Kris.

Reply:
Sure. I can follow up with @KrisBash and we can look to see if all the creates can be updated to PUT in the next version. I believe @wilcobmsft may already have had this conversation with them.  For the current version 2018-11-01-preview apis and given timeline for features, would it make sense to keep the creations consistent as POST.

Reply:
Followed up with Kris offline. For the current preview version, we have to go with POST. we are actively pursuing on using PUT in the next version.